### PR TITLE
Hide partner name on artwork page for benefit and gallery auctions

### PR DESCRIPTION
**Pull request by lilyfromseattle**
For benefit auctions and gallery auctions the partner name and the auction name are the same, so for these auctions we want to hide the subheading. 

__Before:__
Screen Shot 2019-04-30 at 4 40 49 PM

__After:__
Screen Shot 2019-04-30 at 4 39 18 PM

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 6e8d05dabba11667053168a2026390127b75ed53

https://deploy-preview-2355--artsy-reaction.netlify.com

**Comment by samchieng** [comment]
@ds300 good callout! We should keep the alignment in the card, so I'd second using `&nbsp;` or a similar option. Quick mock:

image

**Comment by lilyfromseattle** [comment]
@ds300 When you say add a story do you mean in Jira or do you want me to add a gallery and benefit auction to the artwork page storybooks file?

**Comment by ds300** [comment]
@lilyfromseattle i mean storybooks, yeah 👌

@samchieng thanks for the clarification 🙌

File: src/Components/v2/AuctionCard.tsx [hunk 1]
```diff
@@ -93,11 +93,13 @@ export const LargeAuctionCard = props => (
...
-    {!props.isGalleryAuction && !props.isBenefit && (
-      <Serif size="3t">
-        <Truncator maxLineCount={1}>{props.subHeadline}</Truncator>
-      </Serif>
-    )}
+    <Serif size="3t">
+      <Truncator maxLineCount={1}>
+        {!props.isGalleryAuction && !props.isBenefit
+          ? props.subHeadline
+          : "\u00A0"}
```
Comment (ds300) [inline src/Components/v2/AuctionCard.tsx]: #nonblocking You could change `"\u00A0"` to `<>&nbsp;</>` to make it more readable.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.8.8 :rocket:

**Pull request merged**